src/server/service/kv.rs

```diff
@@ -2106,7 +2108,17 @@ fn needs_reject_raft_append(reject_messages_on_memory_ratio: f64) -> bool {
     let mut usage = 0;
     if memory_usage_reaches_high_water(&mut usage) {
         let raft_msg_usage = (MEMTRACE_RAFT_ENTRIES.sum() + MEMTRACE_RAFT_MESSAGES.sum()) as u64;
-        if raft_msg_usage as f64 > usage as f64 * reject_messages_on_memory_ratio {
+        let cached_entries = get_cached_entries_size();
+        let applying_entries = MEMTRACE_APPLYS.sum() as u64;
+        if (raft_msg_usage + cached_entries + applying_entries) as f64
+            > usage as f64 * reject_messages_on_memory_ratio
+        {
+            info!("need reject log append on memory limit";
```

Comment:
Maybe `warn` is better?

Reply:
done

Reply:
As from minghua's view, we'd better change this to debug.

Reply:
Changed to debug based on discussion comments with @5kbpers